common/web/input-processor/build.sh

```diff
@@ -35,7 +35,7 @@ builder_describe_outputs \
   configure          /node_modules \
   configure:module   /node_modules \
   configure:tools    /node_modules \
-  build:module       build/index.js \
+  build:module       /common/web/input-processor/build/index.js \
```

Comment:
> using absolute (to $KEYMAN_ROOT) rather than relative paths to avoid realpath

Is this only needed on `build` actions and not `configure`?

Reply:
It's for any actions. The `configure` actions here already start with `/`.